lib/js/components/SurveyToggle/SurveyToggle.vue

```diff
@@ -210,7 +210,7 @@ $survey-toggle-colors: (
 	}
 
 	&__label {
-		@include LabelXSBold();
+		@include LabelXSRegular();
```

Comment:
according to https://bethink.atlassian.net/browse/IT-4942 it should be `bold`

Reply:
True, fixed.